workspaces/quay/.changeset/flat-crabs-glow.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage-community/plugin-scaffolder-backend-module-quay': patch
+---
+
+Updated insecure `ui:widget: password` used for token field to `ui:field: Secret`
```

Comment:
These files don't ship with the plugin, I'm not sure that we actually need a changeset for this then. Going to leave that up to @karthikjeeyar  though 👍